augur/metrics/pull_request/pull_request.py

```diff
@@ -427,5 +427,37 @@ def pull_request_acceptance_rate(self, repo_group_id, repo_id=None, begin_date=N
                                                         'begin_date': begin_date, 'end_date': end_date})
         return results
 
+@annotate(tag='pull-request-message')
+def pull_request_message(self, repo_group_id, repo_id=None):
```

Comment:
Please read up the documentation on how to add a new metric. You're missing some keyword arguments.

Reply:
> @mrsaicharan1 I think #651 returns only count for comments, but this pr will give text messages

Umm, you might've been mistaken. PR #651 returns PRs with comments.